lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
Hmm. I'll make this a more practical example:
```
// object 1 is `browserWindow`
add(webContents1, 1) // electron internally grabs `browserWindow` to determine `_isVisible` and `_isMinimized`: https://github.com/atom/electron/blob/master/lib/renderer/override.js#L10-L13
add(webContents1, 1) // actual application grabs `browserWindow` to `openDevTools()`
remove(webContents, 1) // electron has determined `_isVisible` and `_isMinimized`, so `browserWindow` goes out of scope and is garbage collected.
get(webContents1, 1) // actual application is still using `browserWindow`, is going to `closeDevTools()` (but can't because `remove()` deleted reference to `browserWindow`)
```

There's two ways to look at this:
1. `remove()` should _not_ have been called when electron was done finding `_isVisible` and `_isMinimized`. This requires that something else track whether "this reference" is the last reference for that object, and whether or not to call `remove()`
2. `remove()` should always be called. However, since there can be multiple references to an object per `webContents`, there should be a count for each object. That way, when each object is garbage collected, only the last garbage-collect actually deletes the object from `objects-registry`